js/queen.js

```diff
@@ -61,7 +70,11 @@ export class Queen extends Piece{
          * Get col increasing moves
          */
         for (let i = this.colPos + 1; i < maxCols; i++) {
-            if (board[this.rowPos][i] !== undefined && board[this.rowPos][i].getPlayerId() == this.playerId) {
+            if (board[this.rowPos][i] && board[this.rowPos][i].getPlayerId() == this.playerId) {
```

Comment:
`===` should be used instead of `==` on line `73`.

Reply:
Updated.